### ISSUE TITLE
Dependency issue when fetching the package using composer

### ISSUE THREAD

**Issue by Lux589** [stephenlake/AddPay-PHP-SDK#2]
when I run `composer require stephenlake/addpay-php-sdk`

I get the following errors:

```
Using version ^0.8.7 for stephenlake/addpay-php-sdk
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for stephenlake/addpay-php-sdk ^0.8.7 -> satisfiable by stephenlake/addpay-php-sdk[v0.8.7].
    - Conclusion: remove symfony/var-dumper 3.1.x-dev
    - Conclusion: don't install symfony/var-dumper 3.1.x-dev
    - stephenlake/addpay-php-sdk v0.8.7 requires wanfeiyy/dd ^1.0 -> satisfiable by wanfeiyy/dd[1.0.0, 1.0.1, 1.0.2, v1.0.3].
    - wanfeiyy/dd 1.0.0 requires symfony/var-dumper ^3.2 -> satisfiable by symfony/var-dumper[3.2.x-dev, 3.3.x-dev, 3.4.x-dev, v3.2.0, v3.2.0-BETA1, v3.2.0-RC1, v3.2.0-RC2, v3.2.1, v3.2.10, v3.2.11, v3.2.12, v3.2.13, v3.2.14, v3.2.2, v3.2.3, v3.2.4, v3.2.5, v3.2.6, v3.2.7, v3.2.8, v3.2.9, v3.3.0, v3.3.0-BETA1, v3.3.0-RC1, v3.3.1, v3.3.10, v3.3.11, v3.3.12, v3.3.13, v3.3.14, v3.3.15, v3.3.16, v3.3.2, v3.3.3, v3.3.4, v3.3.5, v3.3.6, v3.3.7, v3.3.8, v3.3.9, v3.4.0, v3.4.0-BETA1, v3.4.0-BETA2, v3.4.0-BETA3, v3.4.0-BETA4, v3.4.0-RC1, v3.4.0-RC2, v3.4.1, v3.4.2, v3.4.3, v3.4.4].
    - wanfeiyy/dd 1.0.1 requires symfony/var-dumper ^3.2 -> satisfiable by symfony/var-dumper[3.2.x-dev, 3.3.x-dev, 3.4.x-dev, v3.2.0, v3.2.0-BETA1, v3.2.0-RC1, v3.2.0-RC2, v3.2.1, v3.2.10, v3.2.11, v3.2.12, v3.2.13, v3.2.14, v3.2.2, v3.2.3, v3.2.4, v3.2.5, v3.2.6, v3.2.7, v3.2.8, v3.2.9, v3.3.0, v3.3.0-BETA1, v3.3.0-RC1, v3.3.1, v3.3.10, v3.3.11, v3.3.12, v3.3.13, v3.3.14, v3.3.15, v3.3.16, v3.3.2, v3.3.3, v3.3.4, v3.3.5, v3.3.6, v3.3.7, v3.3.8, v3.3.9, v3.4.0, v3.4.0-BETA1, v3.4.0-BETA2, v3.4.0-BETA3, v3.4.0-BETA4, v3.4.0-RC1, v3.4.0-RC2, v3.4.1, v3.4.2, v3.4.3, v3.4.4].
    - wanfeiyy/dd 1.0.2 requires symfony/var-dumper ^3.2 -> satisfiable by symfony/var-dumper[3.2.x-dev, 3.3.x-dev, 3.4.x-dev, v3.2.0, v3.2.0-BETA1, v3.2.0-RC1, v3.2.0-RC2, v3.2.1, v3.2.10, v3.2.11, v3.2.12, v3.2.13, v3.2.14, v3.2.2, v3.2.3, v3.2.4, v3.2.5, v3.2.6, v3.2.7, v3.2.8, v3.2.9, v3.3.0, v3.3.0-BETA1, v3.3.0-RC1, v3.3.1, v3.3.10, v3.3.11, v3.3.12, v3.3.13, v3.3.14, v3.3.15, v3.3.16, v3.3.2, v3.3.3, v3.3.4, v3.3.5, v3.3.6, v3.3.7, v3.3.8, v3.3.9, v3.4.0, v3.4.0-BETA1, v3.4.0-BETA2, v3.4.0-BETA3, v3.4.0-BETA4, v3.4.0-RC1, v3.4.0-RC2, v3.4.1, v3.4.2, v3.4.3, v3.4.4].
    - wanfeiyy/dd v1.0.3 requires symfony/var-dumper ^3.2 -> satisfiable by symfony/var-dumper[3.2.x-dev, 3.3.x-dev, 3.4.x-dev, v3.2.0, v3.2.0-BETA1, v3.2.0-RC1, v3.2.0-RC2, v3.2.1, v3.2.10, v3.2.11, v3.2.12, v3.2.13, v3.2.14, v3.2.2, v3.2.3, v3.2.4, v3.2.5, v3.2.6, v3.2.7, v3.2.8, v3.2.9, v3.3.0, v3.3.0-BETA1, v3.3.0-RC1, v3.3.1, v3.3.10, v3.3.11, v3.3.12, v3.3.13, v3.3.14, v3.3.15, v3.3.16, v3.3.2, v3.3.3, v3.3.4, v3.3.5, v3.3.6, v3.3.7, v3.3.8, v3.3.9, v3.4.0, v3.4.0-BETA1, v3.4.0-BETA2, v3.4.0-BETA3, v3.4.0-BETA4, v3.4.0-RC1, v3.4.0-RC2, v3.4.1, v3.4.2, v3.4.3, v3.4.4].
    - Can only install one of: symfony/var-dumper[3.2.x-dev, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[3.3.x-dev, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[3.4.x-dev, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.0, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.0-BETA1, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.0-RC1, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.0-RC2, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.1, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.10, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.11, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.12, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.13, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.14, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.2, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.3, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.4, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.5, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.6, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.7, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.8, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.2.9, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.0, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.0-BETA1, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.0-RC1, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.1, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.10, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.11, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.12, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.13, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.14, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.15, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.16, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.2, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.3, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.4, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.5, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.6, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.7, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.8, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.3.9, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.4.0, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.4.0-BETA1, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.4.0-BETA2, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.4.0-BETA3, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.4.0-BETA4, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.4.0-RC1, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.4.0-RC2, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.4.1, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.4.2, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.4.3, 3.1.x-dev].
    - Can only install one of: symfony/var-dumper[v3.4.4, 3.1.x-dev].
    - Installation request for symfony/var-dumper == 3.1.9999999.9999999-dev -> satisfiable by symfony/var-dumper[3.1.x-dev].
```
Been struggling with this for some time please help

**Comment by stephenlake** [comment]
Fixed in [v0.8.9](https://github.com/stephenlake/AddPay-PHP-SDK/tree/v0.8.9).

**Comment by stephenlake** [comment]
Fixed in [v0.8.11](https://github.com/stephenlake/AddPay-PHP-SDK/releases/tag/v0.8.11).

**Comment by stephenlake** [comment]
Tagged as help wanted. Need to test this package across as many PHP frameworks as possible to ensure there are no more conflicts in requirements.

**Comment by Lux589** [comment]
Thanks @stephenlake the issue has been resolved by [v0.8.11 ](https://github.com/stephenlake/AddPay-PHP-SDK/releases/tag/v0.8.11)
